homeassistant/components/media_player/anthemav.py

```diff
@@ -0,0 +1,225 @@
+"""
+Support for Anthem Network Receivers and Processors.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/media_player.anthemav/
+"""
+import logging
+import asyncio
+
+import voluptuous as vol
+
+from homeassistant.components.media_player import (
+    PLATFORM_SCHEMA, SUPPORT_TURN_OFF, SUPPORT_TURN_ON, SUPPORT_SELECT_SOURCE,
+    SUPPORT_VOLUME_MUTE, SUPPORT_VOLUME_SET, MediaPlayerDevice)
+from homeassistant.const import (
+    CONF_NAME, CONF_HOST, CONF_PORT, STATE_OFF, STATE_ON, STATE_UNKNOWN,
+    EVENT_HOMEASSISTANT_STOP, CONF_SCAN_INTERVAL)
+import homeassistant.helpers.config_validation as cv
+
+REQUIREMENTS = ['anthemav>=1.1.3']
```

Comment:
Mark the requirement as `'anthemav==1.1.3'` for stability. The version can be bumped after testing.